lib/govuk_tech_docs/api_reference/renderer.rb

```diff
@@ -0,0 +1,83 @@
+module GovukTechDocs
+  module ApiReference
+    class Renderer
+      def initialize(app, document)
+        @app = app
```

Comment:
`@app` doesn't seem to be used anywhere else... 👀 Potentially we can remove it?

Reply:
Has since been updated in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training-tech-docs/pull/74/commits/6568e7c0aeb96f6956dc0ac72a07fede907210b3.